### PR TITLE
stock market tokens: show player colors

### LINKED ISSUE #3808: Show the color of the company owner *also* in the market (not just on the map)
This will make it easier to see operating order and when we have back to back companies operating and so on

### PR DESCRIPTION
**Pull request by yzemaze**
closes #3808

image

No change if player colors are off – except for a slight move to the left to center tokens horizontally.
image

### CODE REVIEW

File: assets/app/view/game/stock_market.rb [hunk 1]
```diff
@@ -150,6 +151,18 @@ def token_props(corporation, index = nil, num = nil, spacing = nil)
           props[:attrs][:title] = "#{corporation.name} has operated"
           props[:style][:opacity] = '0.6'
           props[:style][:clipPath] = 'polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%)'
+          border = 'dashed'
```
Comment (jenf) [inline assets/app/view/game/stock_market.rb:154]: remember to include Lib::Color

Comment (jenf) [reply assets/app/view/game/stock_market.rb:154]: otherwise LGTM

Comment (yzemaze) [reply assets/app/view/game/stock_market.rb:154]: > remember to include Lib::Color

What for? player_colors is in Lib::Settings, which is already included.

File: assets/app/lib/settings.rb [hunk 2]
```diff
@@ -34,6 +35,7 @@ module Settings
 
     def self.included(base)
       base.needs :user, default: nil, store: true
+      base.send :include, Lib::Color
```
Comment (yzemaze) [inline assets/app/lib/settings.rb:38]: @jenf this should prevent further mishaps and also alleviate some require/include load.

**Review by jenf — APPROVED** [review]

**Pull request merged**